src/clj/game/core/actions.clj

```diff
@@ -205,7 +205,7 @@
   "Use the 'match strength with ice' function of icebreakers."
   [state side args]
   (let [run (:run @state) card (get-card state (:card args))
-        current-ice (when (and run (> (or (:position run) 0) 0)) (get-card state ((get-run-ices state) (dec (:position run)))))
+        current-ice (when (and run (pos? (or (:position run) 0))) (get-card state ((get-run-ices state) (dec (:position run)))))
```

Comment:
you could also lose another nested bracket by using `(:position run 0)`.

Reply:
Oh damn, if you can do that, there's _a lot_ of fixes available.